### ISSUE TITLE
Packaged game: Can't find Blueprint file to extend 

### ISSUE THREAD

**Issue by whiteted** [ncsoft/Unreal.js#71]
[Error in Log.txt](https://github.com/ncsoft/Unreal.js/files/245865/Error.in.Log.txt)
[CreateLink as txt.txt](https://github.com/ncsoft/Unreal.js/files/245868/CreateLink.as.txt.txt)

We've managed to package our project with unrealJS now, and CreateAndLinkAI.js (attached as text) is run when the level loads. It should extend a blueprint (in line 11) and create an object of this extended type, but it fails to find the blueprint, even when I copied the content folder in with said blueprint.
Essentially I just want an object to be able to run the users' JS code, including calls to function members of that object. 
At the very least can I achieve this with "exec"?

p.s. It works fine running the game in the Editor

**Comment by nakosung** [comment]
As I remember there were some discussions related to this topic. Please check out https://github.com/ncsoft/Unreal.js/wiki/Cooking.

**Comment by whiteted** [comment]
Thanks for finding that, that is exactly my problem.

I've changed that line to
    class AIcontrolJS extends Root.ResolveClass('bpControlActorForJS') { ...

where bpControlActorForJS is the name of the Blueprint, but it doesn't work:
Javascript:Error: C:/Users/Whiteted/Documents/PackagedGroupProj/WindowsNoEditor/GroupProj11/Content/Scripts/CreateAndLinkAI.js:14: TypeError: Class extends value undefined is not a function or null
Javascript:Error: TypeError: Class extends value undefined is not a function or null

I've copied in the /Scripts/Unreal/Content.
Also the example in that wiki page is unclear, is the blueprint called 'SomeNice' or 'blueprint'?
